Sources/Common/Store/MessagingPushConfigOptions.swift

```diff
@@ -0,0 +1,21 @@
+public struct MessagingPushConfigOptions {
```

Comment:
I don't think this belongs in `Common` module. It seems like a file best suited for `MessagingPush` module.

Reply:
✅